### ISSUE TITLE
sphinxcontrib-htmlhelp maintenance required

### ISSUE THREAD

**Issue by mgeier** [sphinx-doc/sphinx#11127]
This is a follow-up to #11098, which was about https://github.com/sphinx-doc/sphinxcontrib-applehelp.

A similar problem exists for https://github.com/sphinx-doc/sphinxcontrib-htmlhelp.

```
sphinx.deprecation.RemovedInSphinx80Warning: The alias 'sphinx.util.progress_message' is deprecated, use 'sphinx.util.display.progress_message' instead. Check CHANGES for Sphinx API modifications.
```

**Comment by domdfcoding** [comment]
A related meta-issue: those repositories have their "issues" sections disabled. Is it the intention that issues are filed here for those packages?

**Comment by jnikula** [comment]
Another related meta-issue: Sphinx deprecates interfaces, but depends on external packages that continue to use the deprecated interfaces.

**Comment by mgeier** [comment]
I'm sure that's just an oversight.

I was wondering why this wasn't detected in CI, and I found a potential problem: #11139.

**Comment by AA-Turner** [comment]
Should be fixed with `sphinxcontrib-htmlhelp` 2.0.1.

A
